walkthrough.ipynb

```diff
@@ -22,24 +30,160 @@
    "id": "9e41dab1",
    "metadata": {},
    "source": [
-    "## Platform Setup and Jupyter Walkthrough\n",
+    "## Jupyter Walkthrough\n",
+    "\n",
+    "[Jupyter Notebooks](https://jupyter-notebook.readthedocs.io/en/stable/) are an executable code/text environment for (usually) Python code. Our Jupyter environment is in JupyterHub. To work with Jupyter, click \"run cell\" to run the code and see results below the cell you're currently running. Cells are executed sequentially. \n",
+    "\n",
+    "In some cells, you will see cases where there are variables that you'll need to pre-populate before running the cell. They look like this. The code will not work unless you replace it!  \n",
+    "\n",
+    "```python\n",
+    "# suggestion: \"lumigator_enthusiasts\"\n",
+    "team_name = TEAM_NAME_HERE\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "54bfb102",
+   "metadata": {},
+   "source": [
+    "# Running cells \n",
+    "To run a cell, press the \"play\" icon in the top bar. If a cell is taking too long, you can press stop. \n",
```

Comment:
```suggestion
    "To run a cell, press the \"play\" icon in the top bar (you can also hit Shift+Enter to run and proceed to the following cell).\n",
```
Not sure about the stop thing, do we want to allow people to possibly break things halfway through?